gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=http\://aurora/nexus/service/local/repositories/gradle-distributions/content/gradle-5.5.1-bin.zip
+distributionUrl=http\://aurora/nexus/service/local/repositories/gradle-distributions/content/gradle-5.6-bin.zip
```

Comment:
Kan du hente denne fra nexus.sits.no i steden for. Den er åpen for alle og da slipper jeg å endre på min mac hver gang. 

./gradlew wrapper --gradle-distribution-url=https://nexus.sits.no/repository/gradle-distributions/gradle-5.6-bin.zip